Sources/Mistica/Components/Lists/ListCellContentView.swift

```diff
@@ -276,9 +276,9 @@ open class ListCellContentView: UIView {
         }
     }
 
-    public func setHighlighted(_ highlighted: Bool, animated: Bool) {
-        guard isHighlighted != highlighted else { return }
-        isHighlighted = highlighted
+    public func setHighlighted(_ isHighlighted: Bool, animated: Bool) {
+        guard isHighlighted != isHighlighted else { return }
```

Comment:
I think you have to use `self` to distinguish one from the other, otherwise it is comparing itself 

```suggestion
        guard self.isHighlighted != isHighlighted else { return }
```